backend/pkg/modules/internal/traceanalyzer/weakbasicauth/findings.go

```diff
@@ -241,6 +241,12 @@ func (a APIAnnotationSamePassword) Redacted() utils.TraceAnalyzerAPIAnnotation {
 }
 
 func (a *APIAnnotationSamePassword) ToAPIFinding() oapicommon.APIFinding {
+	var additionalInfo *map[string]interface{}
+	if len(a.APIs) > 0 {
+		additionalInfo = &map[string]interface{}{
+			"apis": a.APIs,
```

Comment:
Are you keeping the list of affected APIs in the description and in the additional Info?

Reply:
Yes.